x-pack/plugins/security_solution/server/ui_settings.ts

```diff
@@ -221,6 +222,19 @@ export const initUiSettings = (
         })
       ),
     },
+    [ENABLE_CCS_READ_WARNING_SETTING]: {
+      name: i18n.translate('xpack.securitySolution.uiSettings.enableCcsReadWarningLabel', {
+        defaultMessage: 'CCS Read Warning',
+      }),
+      value: true,
+      description: i18n.translate('xpack.securitySolution.uiSettings.enableCcsWarningDescription', {
+        defaultMessage: '<p>Enables cross cluster search read warnings</p>',
+      }),
+      type: 'boolean',
+      category: [APP_ID],
+      requiresPageReload: true,
```

Comment:
If all of your logic for checking this setting lives on the server, do you really need to require a page reload?

Reply:
I suppose not, I was just sticking to the pattern there, I'll go ahead and remove that